paasta_tools/mesos_tools.py

```diff
@@ -312,49 +352,45 @@ def status_mesos_tasks_verbose(job_id, get_short_task_id, tail_stdstreams=False)
     """
     output = []
     running_and_active_tasks = get_running_tasks_from_active_frameworks(job_id)
-    output.append("  Running Tasks:")
-    rows_running = [[
+    list_title = "Running Tasks:"
+    table_header = [
         "Mesos Task ID",
         "Host deployed to",
         "Ram",
         "CPU",
         "Deployed at what localtime"
-    ]]
-    for task in running_and_active_tasks:
-        rows_running.append(format_running_mesos_task_row(task, get_short_task_id))
-    tasks_table_running = ["    %s" % row for row in format_table(rows_running)]
-    if not tail_stdstreams:
-        output.extend(tasks_table_running)
-    else:
-        tasks_stdstreams_running = []
-        for task in running_and_active_tasks:
-            tasks_stdstreams_running.append(format_stdstreams_tail_for_task(task, get_short_task_id))
-        output.append(tasks_table_running[0])  # header
-        output.extend(zip_tasks_verbose_output(tasks_table_running[1:], tasks_stdstreams_running))
+    ]
+    output.extend(format_task_list(
+        running_and_active_tasks,
+        list_title,
```

Comment:
Can you do kwargs on both calls on this to make it more obvious for what "False" refers to, etc?